workspace/utilities/com/schema-search.xsl

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+	<xsl:template name="schema-search">
+		<xsl:param name="url" select="'@todo the search page url'" />
```

Comment:
@PascalPiche The todo was a good idea, but it will require us to look to this template to see if it's used.

I first thought it was a good idea, but think it is bad now.

How can we have a safeguard, that only works in dev, which would tell us  that ?

Right now, the best I can think of is this:

1. Create a dev class that makes the `<script>` tag display:block, fixed ?, red background ?
1. Add the class if the url is empty ...

@dominic-blain ? @hotdoy  ?

Reply:
I would do this:
1. Have the default value of url be an empty string
2. Wrap the script tag with a if statement to render the script only when $url is not empty

If we want a todo, let's add it in a comment to catch it on our checklist.